launcher/agent/agent.go

```diff
@@ -115,25 +154,71 @@ func (a *agent) Attest(ctx context.Context, opts AttestAgentOpts) ([]byte, error
 		return nil, fmt.Errorf("failed to get principal tokens: %w", err)
 	}
 
-	var buf bytes.Buffer
-	if err := a.cosCel.EncodeCEL(&buf); err != nil {
-		return nil, err
-	}
-
-	attestation, err := a.attest(challenge.Nonce, buf.Bytes())
-	if err != nil {
-		return nil, fmt.Errorf("failed to attest: %v", err)
-	}
-
 	req := verifier.VerifyAttestationRequest{
 		Challenge:      challenge,
 		GcpCredentials: principalTokens,
-		Attestation:    attestation,
+		Attestation:    nil,
```

Comment:
nit: zero value is nil. you can remove this field.

Reply:
done